spec/features/visitor_views_profile_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "A Visitor views a profile" do
+  scenario "and sees recommendations" do
+    someone_elses_show = create(:television_show)
+    recommendation = create(:recommendation)
+    user = recommendation.user
+    television_show = recommendation.recommendable
+    sign_in(user)
+
+    visit user_path(user)
+
+    expect(page).to have_text(television_show.title)
+    expect(page).not_to have_text(someone_elses_show.title)
```

Comment:
This assertion, to me, either doesn't feel necessary in this spec, or could go into a separate spec.